src/data/report-service.js

```diff
@@ -6,7 +6,7 @@ const ReportService = {
   sendReport(params) {
     const mediaType = params.mediaType ?? IMAGE
     return VersionedApiService.post(
-      `/${getResourceSlug(mediaType)}${params.identifier}/report/`,
+      `/${getResourceSlug(mediaType)}${params.identifier}/report`,
```

Comment:
I'm not sure about removing this slash. The API endpoint has a slash, and `APPEND_SLASH` set to true so the post data will be lost in the redirect.

<img width="1818" alt="Screenshot 2022-03-16 at 11 28 58 AM" src="https://user-images.githubusercontent.com/16580576/158538413-60187ab7-5ed9-4c7c-bebf-8c63fb75715c.png">

Reply:
And when I send the same request to the API, I get 405 Method Not Allowed (presumably because Django went ahead with the redirect and it became a GET request).
<img width="1818" alt="Screenshot 2022-03-16 at 11 31 15 AM" src="https://user-images.githubusercontent.com/16580576/158538894-b43bdf45-a070-4a1c-ac25-9a612baa699e.png">